NavigationAdvancedSample/app/src/main/res/layout/activity_main.xml

```diff
@@ -24,6 +24,7 @@
 
     <androidx.fragment.app.FragmentContainerView
         android:id="@+id/nav_host_container"
+        android:name="androidx.navigation.dynamicfeatures.fragment.DynamicNavHostFragment"
```

Comment:
Deze lijkt mij onnodig. Wat is de funtie ervan?

Staat wel in het github voorbeeld project, maar als ik het weghaal werkt alles nog steeds.

Reply:
Volgens de officieele android documentatie is dit nodig.